### PR TITLE
ADR-1051 UI Tests Added for Declare Ethylene Gas Or Molasses page

### PR DESCRIPTION
**Pull request by Kavitakum**
UI Tests Added for Declare Ethylene Gas Or Molasses page

### CODE REVIEW

File: src/test/resources/features/quarterlySpiritsQuestionsJourney/QuarterlySpiritsQuestionsErrorMessages.feature [hunk 1]
```diff
@@ -88,7 +88,22 @@ Feature: Quarterly Spirits Questions Journey - Error Messages
...
+    When I enter "555.55" for "Beer" on "How Much Alcohol Have You Used Page"
+    And I enter "666.66" for "Wine" on "How Much Alcohol Have You Used Page"
+    And I enter "777.77" for "Made-wine" on "How Much Alcohol Have You Used Page"
+    And I enter "888.88" for "Cider or perry" on "How Much Alcohol Have You Used Page"
+    And I click save and continue button on "How Much Alcohol Have You Used Page"
+    Then I am presented with the "Declare Ethylene Gas Or Molasses Page"
+    And I click save and continue button on "Declare Ethylene Gas Or Molasses Page"
+    Then I am presented with the "Declare Ethylene Gas Or Molasses Page" error page
+    And The error summary title is "<errorMessageHeader>" and the error message is "Enter how much ethylene gas you have used"
+    And The error summary title is "<errorMessageHeader>" and the error message is "Enter how much molasses you have used"
```
Comment (chiransan01) [inline src/test/resources/features/quarterlySpiritsQuestionsJourney/QuarterlySpiritsQuestionsErrorMessages.feature:100]: These come from the same components so only one validation is enough

Comment (chiransan01) [inline src/test/resources/features/quarterlySpiritsQuestionsJourney/QuarterlySpiritsQuestionsErrorMessages.feature:100]: These come from the same components so only one validation is enough

Comment (Kavitakum) [reply src/test/resources/features/quarterlySpiritsQuestionsJourney/QuarterlySpiritsQuestionsErrorMessages.feature:100]: Yes, but in all pages we are validating atleast 1 error message for all fields. In that case we will have to update it for all page

Comment (chiransan01) [reply src/test/resources/features/quarterlySpiritsQuestionsJourney/QuarterlySpiritsQuestionsErrorMessages.feature:100]: One validation per page is fine. I did the same thing with my previous PR. In an ideal scenario, all these validations will be checked in Unit tests but we are adding here to satisfy the audit people

**Review by chiransan01 — APPROVED** [review]

**Pull request merged**